lib/landable/traffic/event_publisher.rb

```diff
@@ -0,0 +1,120 @@
+module Landable
+  module EventPublisher
+    mattr_accessor :page_view, :visit, :event_type, :amqp_configuration
+
+    class << self
+      def publish(page_view, page_view_created_at)
+        return unless service_enabled?
+
+        event_type = event_mapping[page_view.path]
+        if event_type.is_a?(Hash)
+          request_type = page_view.http_method
+          event_type = event_type[request_type]
+        end
+
+        @page_view = page_view
+        @page_view_created_at = page_view_created_at
+        @visit = page_view.visit
+        if event_type
+          @event_type = event_type
+        end
+        messaging_service.publish(message)
+      end
+
+      private
+
+      def config_hash
+        @config_hash = Landable.configuration.amqp_configuration
```

Comment:
I would not hide the meaning of this variable in the name. So instead of ```config_hash```, I would just use ``@ampqq =   Landable.configuration.amqp_configuration``